server/db/migrate/20150216190947_add_rhev_params.rb

```diff
@@ -0,0 +1,16 @@
+class AddRhevParams < ActiveRecord::Migration
+  def change
+    remove_column :fusor_deployments, :rhev_params
+    add_column :fusor_deployments, :rhev_hypervisor_host_id, :integer
```

Comment:
I cleaned up the database:
```
drop table fusor_deployments;
delete from schema_migrations where version = '20150216190947';
delete from schema_migrations where version = '20150123165155';
commit;
```

Then I ran ```rake db:migrate```.

Fusor table is back with the following attributes:

```
katello=# \d fusor_deployments
                                            Table "public.fusor_deployments"
          Column          |            Type             |                           Modifiers                            
--------------------------+-----------------------------+----------------------------------------------------------------
 id                       | integer                     | not null default nextval('fusor_deployments_id_seq'::regclass)
 name                     | character varying(255)      | 
 lifecycle_environment_id | integer                     | 
 organization_id          | integer                     | 
 deploy_rhev              | boolean                     | 
 deploy_cfme              | boolean                     | 
 deploy_openstack         | boolean                     | 
 rhev_hypervisor_host_id  | integer                     | 
 rhev_engine_host_id      | integer                     | 
 rhev_hypervisor_hostname | character varying(255)      | 
 rhev_engine_hostname     | character varying(255)      | 
 rhev_database_name       | character varying(255)      | 
 rhev_cluster_name        | character varying(255)      | 
 rhev_storage_name        | character varying(255)      | 
 rhev_storage_type        | character varying(255)      | 
 rhev_storage_address     | character varying(255)      | 
 rhev_cpu_type            | character varying(255)      | 
 rhev_share_path          | character varying(255)      | 
 cfme_install_loc         | character varying(255)      | 
 created_at               | timestamp without time zone | not null
 updated_at               | timestamp without time zone | not null
Indexes:
    "fusor_deployments_pkey" PRIMARY KEY, btree (id)
```